### ISSUE TITLE
Add way to toggle plushies on/off

### ISSUE THREAD

**Issue by EugeneBeanblossom** [ReillyGregorio/MobScarecrow#18]
Having Plushies is a great idea!  I wanted to try to use the Zombie Plushie to make an iron farm, but the problem is, it's always on as long as it's within an 8 block radius so it never lets the villagers sleep which is required to produce iron golems.  After a while, the iron golems stop appearing.  Do you have any suggestions for how to make this work?  If not, could they be toggled with redstone or something?

**Comment by ArcanePigeon** [comment]
I can add a block that when under a scarecrow disables it in the next update. Then you can just use pistons to swap it out when needed

**Comment by AppleMangoOrange** [comment]
> you can just use pistons to swap it out when needed

Just to help me manually control it with pistons, how big _exactly_ is the 8 block radius? Is it a cube or a sphere? Is it exactly 8? Does it work through blocks? I know this is a lot, but it's necessary to design a consistent farm.

EDIT: From my tinkering it seems to be an 8-block sphere centred at the middle of the plushie. It also seems to work through blocks. Is this correct? If yes, is it possible to add a config to make this a cube?

**Comment by ArcanePigeon** [comment]
yes, it does work through blocks. I probably will not add a config option to change the distance calculation to a cube. but if you can code you are free to add the feature and submit a pull request.
